web/src/routes/Layout/index.tsx

```diff
@@ -327,6 +328,7 @@ class LayoutComponent extends React.Component<LayoutProps, LayoutState> {
     }
 
     const commonRoutes = [
+      <Route key="deeplink" exact path={['/p2p/', '/p2p']} component={DeeplinkScreen} />,
```

Comment:
А давай объединим с BoardScreen. Тогда редирект не нужно делать.
Примерно так:
```suggestion
<Route
  key="BoardScreen"
  path={['/p2p/:filter?', '/:lang/p2p/:filter?']}
  render={(props) => <><DeeplinkScreen {...props} /><BoardScreen {...props} /></>}
/>
```

Reply:
Давай сделаю в отдельном PR где заменю будут ссылки на p2p страницы из baseapp

Reply:
Ок